test/service/dune

```diff
@@ -0,0 +1,21 @@
+(executable
+ (name test)
+ (libraries
+  ocaml_ci
+  alcotest
+  alcotest-lwt
+  ppx_deriving_yojson.runtime
+  ocaml-ci-client-lib
+  timedesc-tzlocal.utc
+  logs.fmt)
+ (preprocess
+  (pps ppx_deriving.eq ppx_deriving_yojson)))
+
+(rule
+ (alias runtest)
+ (package ocaml-ci-service)
+ (deps
+  (source_tree migrations)
+  (package solver-service))
+ (action
+  (run ./test.exe -e)))
```

Comment:
This change of combining alcotest_lwt and plain alcotests into a single executable should be removed from this PR.
It is not necessary for getting solver service integrated.